app/scripts/modules/core/src/presentation/main.less

```diff
@@ -1124,13 +1116,19 @@ textarea.code {
   }
 }
 
+.modal-open {
+  .modal {
+    z-index: var(--layer-critical) !important;
```

Comment:
obligatory "is `!important` really the best option to use here?" question

Reply:
I hate using `!important`, but there was some overrides happening here at the `bootstrap` level.